src/main/java/jp/ac/morijyobi/equipmentmanagementsystem/controller/PasswordChangeController.java

```diff
@@ -0,0 +1,33 @@
+package jp.ac.morijyobi.equipmentmanagementsystem.controller;
+
+import jp.ac.morijyobi.equipmentmanagementsystem.bean.output.ErrorMessage;
+import jp.ac.morijyobi.equipmentmanagementsystem.bean.output.SuccessMessage;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+@Controller
+@RequestMapping("/password_change")
+public class PasswordChangeController {
+    @GetMapping()
+    public String get() {
+        return "password_change";
+    }
+
+    @PostMapping()
+    public String post(final RedirectAttributes redirectAttributes) {
+        final boolean result = true;
+
+        if (result) {
+            final var message = new SuccessMessage("送信しました。");
+            redirectAttributes.addFlashAttribute("message", message);
+        } else {
+            final var message = new ErrorMessage("送信時にエラーが発生しました。");
+            redirectAttributes.addFlashAttribute("message", message);
+        }
+
+        return "redirect:/password_change";
+    }
```

Comment:
早期リターン推奨です。
また、Model や RedirectAttributes に送る際、("message", message) ではなく、(message) だけにする、という話を前していたと思います。

```suggestion
    @PostMapping()
    public String post(final RedirectAttributes redirectAttributes) {
        final boolean result = true;

        if (result) {
            final var message = new SuccessMessage("送信しました。");
            redirectAttributes.addFlashAttribute(message);
            return "redirect:遷移先"
        } 
        
        final var message = new ErrorMessage("送信時にエラーが発生しました。");
        redirectAttributes.addFlashAttribute(message);
        return "redirect:/password_change";
    }
```

Reply:
- https://github.com/saito-kazumasa/equipment-management-system/pull/103/commits/550064675ca9ee6b94f88948f306b9b295ff6da3
- https://github.com/saito-kazumasa/equipment-management-system/pull/103/commits/6db8e3158dfbfbef299467bb873e7efff50a1828
- https://github.com/saito-kazumasa/equipment-management-system/pull/103/commits/e3ab02ff0571b7e1b712de17d74ab9fe3f6db390
  - メッセージの表示を、早期リターンの形に変更しました。
  - 遷移先の画面がない箇所は、遷移先の画面名を明記しました。
  - (message)に変更して、メッセージが表示されなくなりました。

[PasswordResetContoller.java](https://github.com/saito-kazumasa/equipment-management-system/blob/main/src/main/java/jp/ac/morijyobi/equipmentmanagementsystem/controller/PasswordResetController.java)に、("message", message)と(message)があるのですが、このブランチ以外で修正しますか？